src/containers/collectibles/helpers.ts

```diff
@@ -111,20 +123,23 @@ export const getCollectibleImage = async (collectible: Collectible) => {
     imageThumbnailUrl,
     name
   } = collectible
+  if (imageThumbnailUrl?.endsWith('.gif')) {
+    return await getFrameFromGif(imageThumbnailUrl, name || '')
+  }
+  if (imagePreviewUrl?.endsWith('.gif')) {
+    return await getFrameFromGif(imagePreviewUrl, name || '')
+  }
   if (imageUrl?.endsWith('.gif')) {
     return await getFrameFromGif(imageUrl, name || '')
   }
   if (imageOriginalUrl?.endsWith('.gif')) {
     return await getFrameFromGif(imageOriginalUrl, name || '')
   }
-  if (imagePreviewUrl?.endsWith('.gif')) {
-    return await getFrameFromGif(imagePreviewUrl, name || '')
-  }
-  if (imageThumbnailUrl?.endsWith('.gif')) {
-    return await getFrameFromGif(imageThumbnailUrl, name || '')
-  }
   if (imageUrl) {
```

Comment:
maybe we should try other images in case there is no image_url i.e. thumbnail, preview, original?